src/commands/org/login/sfdx-url.ts

```diff
@@ -33,10 +33,18 @@ export default class LoginSfdxUrl extends AuthBaseCommand<AuthFields> {
     'sfdx-url-file': Flags.file({
       char: 'f',
       summary: messages.getMessage('flags.sfdx-url-file.summary'),
-      required: true,
+      required: false,
       deprecateAliases: true,
       aliases: ['sfdxurlfile'],
     }),
+    'sfdx-url-stdin': Flags.file({
```

Comment:
We should add `exclusive: ['sfdx-url-file']` to make sure that people don't provide both

Reply:
@mdonnalley, question:  It sounds to me like `--sfdx-url-stdin` can take only one value (`-`), and if it's present then the user is doing the piping-through-stdin thing.  So wouldn't it make more sense to make `--sfdx-url-stdin` a Boolean flag?  Or maybe I'm missing something.

Reply:
I can provide some context, this is what I based the solution on: https://github.com/oclif/core/issues/761#issue-1851890327

In this case, the value should always be `-` but that is consistent with CLI standards.